src/react-queries/queries/common.ts

```diff
@@ -90,43 +89,15 @@ export const common = createQueryKeys('common', {
   }),
   evmFees: ({
     data,
-    wallet,
```

Comment:
We don't need this anymore since we're only returning a queryKey here, not a queryFn